packages/blueprints/blueprint/src/blueprint.ts

```diff
@@ -142,12 +142,18 @@ export class Blueprint extends Project {
      * copy all non-src file from proposedBundle into the resolved bundle
      * only src is merge constructed.
      */
-    const supersetNonSourcePaths: string[] = filepathSet([proposedBundle], ['**/*', '!src/**']);
-    for (const filepath of supersetNonSourcePaths) {
-      const outputPath = path.join(this.outdir, filepath);
-      fs.mkdirSync(path.dirname(outputPath), { recursive: true });
-      const filecontent = fs.readFileSync(path.join(proposedBundle, filepath));
-      fs.writeFileSync(outputPath, filecontent);
+    const supersetNonSourceNonIssuesPaths: string[] = filepathSet([proposedBundle], ['**/*', '!src/**', '!issues/**']);
```

Comment:
You dont need to update the base blueprint here

Reply:
Just make sure that the proposed bunlde containts the rigt issues

Reply:
resolved through discussion